actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Add `config.action_dispatch.strict_freshness`.
+
+    When set to `true`, the `ETag` header takes precedence over the `Last-Modified` header when both are present,
+    as specificied by RFC 7232, Section 6.
```

Comment:
There is a typo -> specificied -> specified

And just for the reference, the link to RFC https://www.rfc-editor.org/rfc/rfc7232#section-6